lib/Page.js

```diff
@@ -449,21 +422,21 @@ class Page extends EventEmitter {
    * @param {?{username: string, password: string}} credentials
    */
   async authenticate(credentials) {
-    return this._networkManager.authenticate(credentials);
+    return this._frameManager.networkManager().authenticate(credentials);
   }
 
   /**
    * @param {!Object<string, string>} headers
    */
   async setExtraHTTPHeaders(headers) {
-    return this._networkManager.setExtraHTTPHeaders(headers);
```

Comment:
How do you feel about adding `this._networkManager = this._frameMangaer.networkManager()`?

Reply:
Didn't do it to stress the ownership